README.md

```diff
@@ -33,70 +33,92 @@ instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
 ### Run the test suite from your machine
 
-> **You will need to be connected to the VPN to test against Integration or
-> Staging.**
+Read "[Environment variables](#environment-variables)" for an explanation of the environment variables used in the examples below.
 
-The tests require additional configuration to run successfully on a local
-machine.
+*Run Smokey against Production*:
 
 ```sh
-ENVIRONMENT=integration \
+ENVIRONMENT=production
+
 SIGNON_EMAIL="<email-address>" \
 SIGNON_PASSWORD="<password>" \
-bundle exec cucumber
+bundle exec cucumber --tags="not @not$ENVIRONMENT"
```

Comment:
Could avoid putting this on the user by making the default in config/cucumber.yml. (It [supports ERB](https://cucumber.io/docs/cucumber/configuration/?lang=ruby).)

Reply:
Thanks, I'll give profiles more thought in a future PR. Have pushed up a commit to tidy up these examples.